examples/webpack-example/.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "breakConfig": true,
-  "stage": 1,
+  "presets": ["es2015", "stage-1", "react"]
```

Comment:
That's what was used before so that's what I set it as however I've installed babel-preset-stage-0 which means all the stages are available, from 2 to 0 so feel free to modify this as necessary.